VideoApi/VideoApi.UnitTests/Mappings/ParticipantToDetailsResponseMapperTests.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using FluentAssertions;
+using NUnit.Framework;
+using Testing.Common.Helper.Builders.Domain;
+using VideoApi.Domain;
+using VideoApi.Domain.Enums;
+using VideoApi.Mappings;
+
+namespace VideoApi.UnitTests.Mappings
+{
+    public class ParticipantToDetailsResponseMapperTests
+    {
+        [Test]
+        public void should_map_participant_to_response_without_interpreter_room()
+        {
+            var participant = new ParticipantBuilder().WithUserRole(UserRole.Individual).Build();
+
+            var response = ParticipantToDetailsResponseMapper.MapParticipantToResponse(participant);
+
+            response.Should().BeEquivalentTo(participant, options => options
+                .Excluding(x => x.ParticipantRefId)
```

Comment:
didnt know u could do this :)

Reply:
yeah nice little trick and helps when you may have forgotten a property in the target to :)